### DISCUSSION TITLE
Monthly ESMValTool meeting April 2022

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#23] [Meetings]
The next monthly ESMValTool meeting will be in April. The result of the poll is that the meeting will take place on

Tuesday, April 12 at 14:00 CEST (13:00 BST).

If none of the proposed times work for you, please let us know by commenting below.

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

@ESMValGroup/esmvaltool-developmentteam

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we can decide to talk about specific topics with the entire group or make smaller breakout groups to continue the discussion there.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the March meeting can be found here: https://github.com/ESMValGroup/Community/discussions/18#discussioncomment-2332138

**Comment by valeriupredoi** [comment]
Dis happening? 5 more minutes and am bailing for lunch :hamburger:

**Reply by bsolino** [reply]
Is there anybody who can organize a different call? The options we have available here are not the best, but they may do...

**Reply by valeriupredoi** [reply]
maybe we should reschedule - it don't seem we are a full complement anyway - Easter's about to happen :egg:

**Reply by valeriupredoi** [reply]
we'll post the minutes from this call soon :rofl:

**Reply by bouweandela** [reply]
My sincere apologies everyone, I forgot to start the call, again! :cry: I'll make sure I remember and arrange for a backup host next time.
